.github/workflows/helm.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run this workflow on new tags
+name: Publish Helm Chart
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  update-helm-charts:
+    name: Update gh-pages branch helm charts and index
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Helm
+      uses: azure/setup-helm@v4.2.0
+      with:
+        version: 3.14.4
+
+    - name: Check out repo
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Update helm index
+      env:
+        VERSION: ${{ github.event.release.tag_name }}
+        HELM_REPO_PATH: releases/helm-${{ github.event.release.tag_name }}/
+        DOWNLOAD_URL: ${{ join(github.event.release.assets.*.browser_download_url, ' ') }}
+      run: |
+        git config user.name "Github Actions"
+        git config user.email "no-reply@github.com"
```

Comment:
Question (not a blocker): can this be set in the action somehow?

Reply:
you mean like as an argument to the action? 
The documentation and examples I have found all play it like this.